CONTRIBUTING.md

```diff
@@ -96,7 +96,13 @@ Before you submit a pull request, check that it meets these guidelines:
     Put your new functionality into a function with a docstring, and add
     the feature to the list in README.md.
 3.  The pull request should work for Python 2.7+ and 3.6+.
-4.  New changes should be added to CHANGELOG.md
+4.  The project version should be updated accordingly
```

Comment:
We intentionally removed this step from this doc before as we don't want the contributors to do this but only the project owners

Reply:
the version update?
Apologies I was not aware and was always flipping back and forth between repo instructions
will remove.